types/prosemirror-test-builder/index.d.ts

```diff
@@ -32,7 +32,8 @@ type TestNodesUnion =
     | 'hard_break';
 type TestMarksUnion = 'a' | 'link' | 'em' | 'strong' | 'code';
 
-type Args = Array<string | prosemirrorTestBuilder.TaggedProsemirrorNode | prosemirrorTestBuilder.TaggedFlatObject>;
+type Args = Array<string | prosemirrorTestBuilder.TaggedProsemirrorNode | prosemirrorTestBuilder.TaggedFlatObject |
+    object>;
```

Comment:
I'm not a user, from README this can be just:
```ts
type Args = Array<
    | string
    | prosemirrorTestBuilder.TaggedProsemirrorNode
    | prosemirrorTestBuilder.TaggedFlatObject
    | {
          level: 1 | 2 | 3;
      }
>;
```

Reply:
The args parameter can be any kind of object, containing arbitrary fields.